### DISCUSSION TITLE
Downgrading didn't work. Need help

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1089] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was trying to downgrade to 105, but it said it was out of date. I even tried turning off verified mode, but it said it was required by device policy to recover.

**Comment by AshtonDavies** [comment, marked as answer]
v105 is too outdated for a v120+ user.

Administrators do have an option that requires users to recover with IT so they can track who exactly is doing it. I do not know of a way to bypass it, so you should check TN.

**Reply by FunnyCorgi** [reply]
@AshtonDavies how about 119/118?

**Reply by FunnyCorgi** [reply]
@AshtonDavies i saw on the discord that 112 is the lowest downgrade version for kernver 2. does this still apply?
